diff --git a/openfe/tests/protocols/test_openmm_equil_rfe_protocols.py b/openfe/tests/protocols/test_openmm_equil_rfe_protocols.py
index 41065c17..de53a58c 100644
--- a/openfe/tests/protocols/test_openmm_equil_rfe_protocols.py
+++ b/openfe/tests/protocols/test_openmm_equil_rfe_protocols.py
@@ -244,32 +244,7 @@ def test_dry_run_gaff_vacuum(benzene_vacuum_system, toluene_vacuum_system,
         mapping=benzene_to_toluene_mapping,
     )
     unit = list(dag.protocol_units)[0]
-
-    # If we do a lot of GAFF testing, this should be refactored so we don't
-    # have to copy it all over the place.
-    # https://github.com/OpenFreeEnergy/openfe/pull/847#issuecomment-2096810453
-
-
-    import openmmforcefields
-    from packaging import version
-
-    ommff_version = openmmforcefields.__version__
-
-    gaff_should_fail = version.parse(
-        ommff_version
-    ) >= version.parse("0.13.0")
-
-    if gaff_should_fail:
-        from openmmforcefields.generators.template_generators import (
-            GAFFNotSupportedError,
-        )
-
-        with pytest.raises(GAFFNotSupportedError):
-            with tmpdir.as_cwd():
-                sampler = unit.run(dry=True)["debug"]["sampler"]
-    else:
-        with tmpdir.as_cwd():
-            sampler = unit.run(dry=True)["debug"]["sampler"]
+    sampler = unit.run(dry=True)["debug"]["sampler"]
 
 
 @pytest.mark.slow
@@ -1633,7 +1608,7 @@ class TestProtocolResult:
 def test_get_charge_difference(mapping_name, result, request):
     mapping = request.getfixturevalue(mapping_name)
     if result != 0:
-        ion = 'Na\+' if result == -1 else 'Cl\-'
+        ion = r'Na\+' if result == -1 else r'Cl\-'
         wmsg = (f"A charge difference of {result} is observed "
                 "between the end states. This will be addressed by "
                 f"transforming a water into a {ion} ion")
diff --git a/openfe/tests/protocols/test_openmm_plain_md_protocols.py b/openfe/tests/protocols/test_openmm_plain_md_protocols.py
index 566cd982..82088815 100644
--- a/openfe/tests/protocols/test_openmm_plain_md_protocols.py
+++ b/openfe/tests/protocols/test_openmm_plain_md_protocols.py
@@ -176,31 +176,7 @@ def test_dry_run_gaff_vacuum(benzene_vacuum_system, tmpdir):
         mapping=None,
     )
     unit = list(dag.protocol_units)[0]
-
-    # If we do a lot of GAFF testing, this should be refactored so we don't
-    # have to copy it all over the place.
-    # https://github.com/OpenFreeEnergy/openfe/pull/847#issuecomment-2096810453
-
-    import openmmforcefields
-    from packaging import version
-
-    ommff_version = openmmforcefields.__version__
-
-    gaff_should_fail = version.parse(
-        ommff_version
-    ) >= version.parse("0.13.0")
-
-    if gaff_should_fail:
-        from openmmforcefields.generators.template_generators import (
-            GAFFNotSupportedError,
-        )
-
-        with pytest.raises(GAFFNotSupportedError):
-            with tmpdir.as_cwd():
-                system = unit.run(dry=True)["debug"]["system"]
-    else:
-        with tmpdir.as_cwd():
-            system = unit.run(dry=True)["debug"]["system"]
+    system = unit.run(dry=True)["debug"]["system"]
 
 
 @pytest.mark.parametrize('method, backend, ref_key', [
